docs/cases/cases-manage.asciidoc

```diff
@@ -0,0 +1,173 @@
+. After you've finished creating your visualization, click *Save and return* to go back to your case.
+. Click *Preview* to show how the visualization will appear in the case comment.
+. Click *Add Comment* to add the visualization to your case. 
+
+NOTE: Once a visualization has been added to a case, it cannot be modified or deleted. However, you can interact with the visualization by clicking the *Open Visualization* option in the comment menu.
+  
+[role="screenshot"]
+image::images/cases-open-vis.png[Shows where the Open Visualization option is]
+
+[float]
+[[cases-export-import]]
+== Export and import cases
+
+Cases can be <<cases-export, exported>> and <<cases-import, imported>> as saved objects through the Kibana {kibana-ref}/managing-saved-objects.html[Saved Objects] UI.
+
+IMPORTANT: Before importing Lens visualizations, Timelines, or alerts into a space, ensure their data is present. Without it, they won't work after being imported.
+
+[float]
+[[cases-export]]
+=== Export a case
+Use the *Export* option to move cases between different Kibana instances. When you export a case, the following data is exported to a newline-delimited JSON (`.ndjson`) file: case details, user actions, text string comments, case alerts, and lens visualizations (which are exported as JSON blobs).
+
+To export a case:
+
+. Open the main menu, click *Stack Management -> Kibana*, then select the *Saved Objects* tab.
+. Search for the case by choosing a saved object type or entering the case title in the search bar.
+. Select one or more cases, then click the *Export* button.
+. Click *Export*. A confirmation message that your file is downloading displays.
+
++
+TIP: Keep the *Include related objects* option enabled to ensure connectors are exported too.
+
+[role="screenshot"]
+image::images/cases-export-button.png[Shows the export saved objects workflow]
+
+[float]
+[[cases-import]]
+=== Import a case
+
+To import a case:
+
+. Open the main menu, click *Stack Management -> Kibana* and then select the *Saved Objects* tab.
+. Click *Import*.
+. Select the NDJSON file containing the exported case and configure the import options.
+. Click *Import*.
+. Review the import log and click *Done*.
++
+[IMPORTANT]
+=========================
+
+Be mindful of the following:
+
+* If the imported case had connectors attached to it, you'll be prompted to re-authenticate the connectors. To do so, click *Go to connectors* on the *Import saved objects* flyout and complete the necessary steps. Alternatively, open the main menu, then go to *Stack Management -> Alerts and Insights -> Rules and Connectors -> Connectors* to access connectors.
+* If the imported case had attached alerts, verify that the alerts’ source documents are present in the environment. Case features that interact with alerts (such as the Alert Details flyout and rule details page) rely on the alerts’ source documents to function.
```

Comment:
Not sure why Alert Details is capitalized when rule details is not

Reply:
The "d" in "Alert details flyout" should actually be lowercase. A while back, the alert flyout was named "Alert details"  so capitlizing the "d" is a convention from those days. I'm open to revisiting this guideline that at any time though, especially since it's been a few releases since that change. The term "rule details" has always been lowercase as far as I remember. I've noticed some inconsistencies in our other docs, so that's something we might want to address at a later time.